Formula/kubeless.rb

```diff
@@ -0,0 +1,35 @@
+class Kubeless < Formula
+  desc "Kubernetes Native Serverless Framework"
+  homepage "https://github.com/kubeless/kubeless"
+  url "https://github.com/kubeless/kubeless.git",
+    :tag => "v0.2.3",
```

Comment:
@ilovezfs What style of indent are we using here these days? 🙈

Reply:
two spaces is the usual indent increment, I can make it 4 spaces to match with the previous line that seems pretty common as well

Reply:
Yeah, my feeling is that we've been using 4 on URLs specifically because it lines up neatly, but I'm not sure how consistently that's being enforced or whether it's being enforced at all really.